<launch>
  <!-- Run stereo_image_proc -->
  <arg name="folder_path" default="/home/paolo/catkin_ws"/>
  <arg name="stereo_config_path" default="/home/renzo/catkin_ws/src/apple_fields_ignition/fields_ignition/config/stereoParamsSim2Blocks.yaml"/>

  <group ns="/costar_husky_sensor_config_1">
    <node pkg="stereo_image_proc" type="stereo_image_proc" name="stereo_image_proc" output="screen">
      <rosparam command="load" file="$(arg stereo_config_path)"/>
    </node>
  </group>

  <!-- Run script to save the images -->
  <!-- use remap instead of the namespace -->
  <env name="ROS_NAMESPACE" value="costar_husky_sensor_config_1" />
  <node pkg="fields_ignition" type="save_disparity_and_rgb.py" name="save_disparity_and_rgb" args="$(arg folder_path)" output="screen"/>
</launch>

<!-- Ejemplo de ejecucion -->
<!-- Parado en la carpeta de catkin_ws y con la simulacion corriendo (roslaunch fields_ignition fields_ignition.launch) -->
<!-- roslaunch fields_ignition stereo_simulation.launch folder_path:=/home/paolo/catkin_ws -->
